### DISCUSSION TITLE
Is there a way I can auto-add `@types` packages for dependencies?

### DISCUSSION THREAD

**Discussion by abh1kg** [pnpm/pnpm#5839] [Q&A] [answered]
I would like to auto-install `@types` packages for my dependencies whenever possible. `yarn` has a plugin called `types` which does this - I was wondering if there's a way to do this for pnpm as well.

**Comment by zkochan** [comment, marked as answer]
There is no such feature yet. There is a PR to add this feature though. But it is not ready: https://github.com/pnpm/pnpm/pull/4166
